libs/pulumi/src/executors/up/executor.ts

```diff
@@ -14,7 +14,7 @@ export default async function runUpExecutor(
     }
 
     const infrastructureProject =
-        options.infrastructureProject ?? `${context.projectName}-infrastructure`
+        options.infrastructureProject ?? context.projectName
```

Comment:
We could probably fall back to the old behaviour here...

Eg `hello-world` and `hello-world-infrastructure`. We could try `${context.projectName}-infrastructure` first, then if it doesn't exist, fall back to `context.projectName`.

This would work as long as there isn't a project `hello-world-infrastructure-infrastructure`

Reply:
Nah, I don't think it's worth the complexity